### PR TITLE
OCP-25676

### PR DESCRIPTION
**Pull request by weliang1**
Remove a extra space in this case.

@zhaozhanqi @huiran0826 @anuragthehatter @rbbratta
Please help to review, thanks!

### CODE REVIEW

File: features/networking/multus.feature [hunk 1]
```diff
@@ -826,28 +826,24 @@ Feature: Multus-CNI related scenarios
...
+      | n | <%= project.name %>                                                                                                           |
     Then the step should succeed
     And the pod named "runtimeconfig-pod" becomes ready
 
     # Check created pod has correct MAC and IP for interface net1
     When I execute on the pod:
       | /usr/sbin/ip | -d | link |
-    Then the output should contain: 
+    Then the output should contain:
       | net1                 |
```
Comment (anuragthehatter) [inline features/networking/multus.feature]: Indentation

File: features/networking/multus.feature [hunk 2]
```diff
@@ -826,28 +826,24 @@ Feature: Multus-CNI related scenarios
 
     # Create the net-attach-def via cluster admin
     Given I have a project
-    And evaluation of `project.name` is stored in the :project_name clipboard
     When I run the :create admin command with:
-      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/runtimeconfig-def-ipandmac.yaml |
-      | n | <%= cb.project_name %>                                                                                                 |
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/runtimeconfig-def-ipandmac.yaml                                                                                                                           |
```
Comment (anuragthehatter) [inline features/networking/multus.feature]: Indentation

File: features/networking/multus.feature (hunk 1 above)
Comment (anuragthehatter) [inline features/networking/multus.feature]: Indentation

Comment (weliang1) [reply features/networking/multus.feature]: Fixed

File: features/networking/multus.feature (hunk 2 above)
Comment (weliang1) [reply features/networking/multus.feature]: Fixed, thanks!

**Pull request merged**